app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/samples.phtml

```diff
@@ -9,18 +9,19 @@
 <?php
```

Comment:
Please remove not needed closing and opening PHP tags

Reply:
fixed